test/suites/init_preseed.sh

```diff
@@ -0,0 +1,60 @@
+test_init_preseed() {
+  # - lxd init --preseed
+  LXD_INIT_DIR=$(mktemp -d -p "${TEST_DIR}" XXX)
+  chmod +x "${LXD_INIT_DIR}"
+  spawn_lxd "${LXD_INIT_DIR}" false
+  (
+    set -e
+    # shellcheck disable=SC2034
+    LXD_DIR=${LXD_INIT_DIR}
+
+    cat <<EOF | lxd init --preseed
+config:
+  core.https_address: 127.0.0.1:9999
+  images.auto_update_interval: 15
+pools:
```

Comment:
Done.

However I think "storage-pools" might be a tad more appropriate than "storage_pools", since the format and keys of this YAML closely matches the RESTful API, and the endpoint for storage pools is called "/storage-pools" there, not "storage_pools".